grid/social-media/upload-from-url/profile.supr

```diff
@@ -32,14 +32,19 @@ usecase RegisterUpload unsafe {
     url! string!
 
     """
-    Type
+    Media Type
     Media type of the asset
     """
-    type enum {
+    mediaType enum {
```

Comment:
Hmm, media type is another name for MIME type, so having this as enum can be somewhat misleading. But I don't have much better name for this (content type also smells of mime given the http header). How about mediaKind or contentKind?